src/mame/konami/thunderx.cpp

```diff
@@ -384,8 +382,7 @@ void thunderx_state::run_collisions( int s0, int e0, int s1, int e1, int cm, int
 		t0 = p0[4] - p0[2];
 		b0 = p0[4] + p0[2];
 
-		p1 = &m_pmcram[16 + 5 * s1];
-		for (jj = s1; jj < e1; jj++,p1 += 5)
+		for ( p1 = &m_pmcram[s1]; p1 < &m_pmcram[e1]; p1 += 5)
```

Comment:
Also, as above, use “English” spacing around parentheses, and you may as well reduce the scope of `p1` while you’re changing the code anyway.